api/internal/tests/views/test_coverage_viewset.py

```diff
@@ -333,24 +333,6 @@ def test_tree_not_found_for_components(
         res = self._tree(components="Does_not_exist")
         assert res.status_code == 404
 
-    @patch("shared.reports.api_report_service.build_report_from_commit")
-    @patch("services.components.commit_components")
-    def test_tree_not_found_for_components(
```

Comment:
This test looked to be an exact copy of the one above it

Reply:
good catch